### ISSUE TITLE
Run image_demo.py but the visualized_result picture does not have any frame 

### ISSUE THREAD

**Issue by wei-miao** [Jackluisus/VisDrone-ensemble-tensorflow#1]
change cfgs.py:
SAVED_WEIGHTS_DIR = "./saved_weights/single-scale training/r101/"
PRETRAIN_BACKBONE_WEIGHTS = r"./data/pretrained_weights/resnet101_v1d.ckpt"
PRETRAIN_BACKBONE_NAME = 'resnet101_v1d' #resnet50_v1d, resnet101_v1d

**Comment by Jackluisus** [comment]
你预测的是什么图片？如果是无人机的图片，你可以试试加载一个具体的训练好的模型，你看models/meta_architectures.py里面第221行，下面的checkpoint_path换成你的路径，别忘了去掉注释。

**Comment by wei-miao** [comment]
方便加个联系方式吗 微信：woshixiqoxuesheng

**Comment by wei-miao** [comment]
您好，我debug测试了一下，
修改代码  ./core/config/cfgs.py ：
# eval
INFERENCE_SCORE_THRSHOLD = 0.1
IMG_INFERENCE_DIR = r'./data/VisDrone2018-DET-val/images'
MULTI_SCALSE_TESTING=True
NORMALIZED_RESULTS_FOR_MODEL_ENSEMBLE = False
并且修改 ./model/meta_architectures.py:
checkpoint_path = r'./saved_weights/multi-scale-training/r101/visdrone_resnet101_v1d_88000_model.ckpt'

加载的训练模型是多尺度的，测试的时候也用的多尺度，但是我的visualized_result的结果图片上没有框，您方便帮我看看是哪儿出的问题吗？谢谢，如果方便的话，恳请您加一下我的联系方式，非常感谢您。
